cockroachcloud/create-an-account.md

```diff
@@ -5,29 +5,35 @@ toc: true
 docs_area: deploy
 ---
 
-Before you [create a {{ site.data.products.db }} cluster](create-your-cluster.html), you must first create a {{ site.data.products.db }} account. You can register for {{ site.data.products.db }} using a GitHub account or an email address.
+Before you [create a {{ site.data.products.db }} cluster](create-your-cluster.html), you must first create a {{ site.data.products.db }} account. You can register for {{ site.data.products.db }} using a GitHub, Google, or Microsoft account or an email address and password.
+
+{% include cockroachcloud/prefer-sso.md %}
 
 ## Register a new account
 
 To register a new account, navigate to the [{{ site.data.products.db }} registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account).
 
+### Choose an authentication method
+{{site.data.alerts.callout_info}}
+You may authenticate with Cockroach Cloud using an email and password, or with [Single Sign-on (SSO)](cloud-sso.html). Cockroach Cloud supports SSO provided by Github, Google, and Microsoft.
```

Comment:
On this page, also check on the product naming and use the liquid templating as per the style guide.

Reply:
And cap the H in GitHub here